app/services/most_recent_content.rb

```diff
@@ -0,0 +1,40 @@
+require 'gds_api/rummager'
+
+class MostRecentContent
+  include RummagerFields
+
+  attr_reader :content_ids, :filter_content_purpose_supergroup, :number_of_links
+
+  def initialize(content_ids:, filter_content_purpose_supergroup:, number_of_links: 5)
```

Comment:
As it's being displayed in an element that shows three to a row, I imagine at some point we might want to change this to being a multiple of 3? Or we remove the default and make it an explicit choice every time it's displayed?

Reply:
5 is ok for now, as we will be showing the 5 most recent documents across 2 components - 3 in the highlight boxes and 2 in the document list. But we retrieve the 5 docs in one call to rummager 🙂